crates/sui-bridge/src/node.rs

```diff
@@ -37,12 +38,21 @@ pub async fn run_bridge_node(
     prometheus_registry: prometheus::Registry,
 ) -> anyhow::Result<JoinHandle<()>> {
     init_all_struct_tags();
-    let metrics = Arc::new(BridgeMetrics::new(&prometheus_registry));
-    let (server_config, client_config) = config.validate(metrics.clone()).await?;
+    let bridge_metrics = Arc::new(BridgeMetrics::new(&prometheus_registry));
+    let tc_metrics = TrafficControllerMetrics::new(&prometheus_registry);
+
+    let (server_config, client_config) = config.validate(bridge_metrics.clone()).await?;
+    let traffic_controller = server_config.policy_config.map(|policy_config| {
+        Arc::new(TrafficController::spawn(
```

Comment:
in a future PR, we will decide whether to `spawn` or `init` based on the actual acl policy?

Reply:
Yep, once I merge and rebase on the other PR which introduces the spawn/init, this logic will no longer be valid since `spawn` should no longer be public, and will need to be replaced with `init`, which will do the right thing based on the policy